spec/index.adoc

```diff
@@ -344,6 +344,44 @@ where `signing_canister_id` is the id of the signing canister and `reconstruct`
 where `s` is the SHA-256 hash of the `seed` used in the public key and `m` is the SHA-256 hash of the payload.
 --
 
+[#supplementary-technologies]
+== Supplementary Technologies
+
+[#cbor]
+=== CBOR
+
+https://www.rfc-editor.org/rfc/rfc8949[Concise Binary Object Representation(CBOR)] is a data format with a small code footprint, small message size and an extensible interface. CBOR is used extensively throughout the Internet Computer as the primary format for data exchange between components within the system.
+
+https://cbor.io[cbor.io] and  https://en.wikipedia.org/wiki/CBOR[wikipedia.org] contain a lot of helpful background information and relevant tools. https://cbor.me[cbor.me] in particular, is very helpful for converting between CBOR hex and diagnostic information.
+
+For example, the following CBOR hex:
+....
+D9 D9 F7 A3 67 63 6F 6E 74 65 6E 74 A4 6C 72 65 71 75 65 73 74 5F 74 79 70 65 64 63 61 6C 6C 6B 63 61 6E 69 73 74 65 72 5F 69 64 43 AB CD 01 6B 6D 65 74 68 6F 64 5F 6E 61 6D 65 69 73 61 79 5F 68 65 6C 6C 6F 63 61 72 67 48 00 61 73 6D 01 00 00 00 6A 73 65 6E 64 65 72 5F 73 69 67 44 DE AD BE EF 6D 73 65 6E 64 65 72 5F 70 75 62 6B 65 79 58 20 B7 A3 C1 2D C0 C8 C7 48 AB 07 52 5B 70 11 22 B8 8B D7 8F 60 0C 76 34 2D 27 F2 5E 5F 92 44 4C DE
+....
+
+Can be converted into the following CBOR diagnostic format:
+....
+55799({
+  "content": {
+    "request_type": "call",
+    "canister_id": h'ABCD01',
+    "method_name": "say_hello",
+    "arg": h'0061736D01000000'
+  },
+  "sender_sig": h'DEADBEEF',
+  "sender_pubkey": h'B7A3C12DC0C8C748AB07525B701122B88BD78F600C76342D27F25E5F92444CDE'
+})
```

Comment:
The other fields (arg, sender_sig, sender_pubkey) are not valid either (I suppose). Do we want this example to be a realistic one?

Reply:
I copied the example from the "CBOR encoding of requests and responses" section. I don't think the example necessarily needs to be realistic, but I also don't think it should be confusing or misleading. The only thing I want to demonstrate here is the before/after of decoding using cbor.me. Maybe it's more sensible to use a basic example from the CBOR spec: https://www.rfc-editor.org/rfc/rfc8949.html#name-examples-of-encoded-cbor-da

I think `82 61 61 a1 61 62 61 63` -> `["a", {"b": "c"}]` could be a nice example, what do you think?

Reply:
I'd prefer an unrelated example in that case (if we do not make the request a proper one).

Reply:
I've replaced it with the generic example now, thank you